app/Traits/ClearsGlobalSearchCache.php

```
@@ -0,0 +1,53 @@
+
+use App\Livewire\GlobalSearch;
+
+trait ClearsGlobalSearchCache
+{
+    protected static function bootClearsGlobalSearchCache()
+    {
+        static::saved(function ($model) {
+            // Clear search cache when model is saved
+            $teamId = $model->getTeamIdForCache();
+            if (filled($teamId)) {
+                GlobalSearch::clearTeamCache($teamId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Livewire\GlobalSearch' in method 'bootClearsGlobalSearchCache'. (undefined)

(StaticAccess)